### ISSUE TITLE
Does it still support iOS 10 ?

### ISSUE THREAD

**Issue by haithngn** [jianstm/Schedule#16]
Hi @jianstm ,
I have built Schedule via Carthage (Xcode 9.4.1) then integrate to a project and they said:
Module file's minimum deployment target is ios11.4 v11.4: /Users/hai/Frameworks/Schedule.framework/Modules/Schedule.swiftmodule/x86_64.swiftmodule

I have checked Schedule's targets they are setting from 10.10, could you investigate this ?

**Comment by haithngn** [comment]
I have tried on branch 1.x The Carthage said:
Task failed with exit code 65

**Comment by jianstm** [comment]
Hi, @haithngn, sorry for the late reply. I noticed that carthage issue. Schedule was originally created using swift package manager, which doesn't have platforms specified. I think maybe this is the cause of the problem. 
I already added framework targets(iOS, macOS, tvOS, watchOS) to Schedule in branch 1.x, but i haven't tested them yet. Will probably release the tested new version this noon. Sorry about the confusion!
